src/robusta/core/sinks/robusta/rrm/types.py

```diff
@@ -0,0 +1,107 @@
+from datetime import datetime
+from enum import Enum
+from typing import List, Optional
+from pydantic import BaseModel
+
+
+class AccountResourceStatusInfo(BaseModel):
+    error: Optional[str]
+
+
+class AccountResourceStatusType(str, Enum):
+    error = "error"
+    success = "success"
+
+
+class AccountResourceStatus(BaseModel):
+    account_id: str
+    cluster_id: str
+    status: Optional[AccountResourceStatusType]
+    info: Optional[AccountResourceStatusInfo]
+    synced_revision: Optional[datetime]
+    latest_revision: Optional[datetime]
+    updated_at: datetime
+
+
+class ResourceKind(str, Enum):
+    PrometheusAlert = "PrometheusAlert"
+
+
+class AccountResource(BaseModel):
+    entity_id: str
+    resource_kind: ResourceKind
+    clusters_target_set: Optional[List[str]] = None
+    resource_state: Optional[dict] = None
+    deleted: bool = False
+    enabled: bool = True
+    updated_at: datetime
+
+
+class ResourceState(BaseModel):
+    pass
+
+
+class PrometheusAlertAnnotations(BaseModel):
```

Comment:
What is the types of `annotations` and `labels` on the DB?
Aren't these just dictionaries?

Why do we have a typed classes, with a subset of the fields?